soccer/RobotConfig.hpp

```diff
@@ -78,11 +84,14 @@ class RobotConfig {
  */
 class RobotStatus {
 public:
+    RobotStatus() {}
     RobotStatus(Configuration* config, QString prefix);
-    ~RobotStatus() {}
 
-    ConfigBool* chipper_enabled;
-    ConfigBool* kicker_enabled;
-    ConfigBool* ball_sense_enabled;
-    ConfigBool* dribbler_enabled;
+    RobotStatus(RobotStatus&&) = default;
+    RobotStatus& operator=(RobotStatus&&) = default;
```

Comment:
Are we sure we want to allow copies with all the unique ptrs here?

Reply:
That's a move. Moving is okay, copying isn't.

Reply:
Oof, super misread that